h/views/badge.py

```diff
@@ -70,13 +70,22 @@ def badge(request):
         cache_control.public = True
         cache_control.max_age = 86400  # 1 day
 
+        # `pyramid_authsanity` sets a response callback which adds Vary=Cookie
+        # which will totally break our caching. To get around this we add
+        # another callback which should be called after to disable it.
```

Comment:
Perhaps worth noting here that `pyramid_authsanity`'s behavior is correct most of the time because any access to `request.authenticated_userid` implies that the response is user-specific and therefore not publicly cacheable. We're overriding that here with higher-level knowledge.